libs/sdk-ui-ext/src/dashboardView/DashboardRenderer.tsx

```diff
@@ -177,11 +180,16 @@ export const DashboardRenderer: React.FC<IDashboardRendererProps> = memo(functio
                 }
 
                 const currentSize = item.size()[screen];
+
                 const minHeight =
-                    !currentSize.gridHeight && !currentSize.heightAsRatio
+                    getDashboardLayoutHeight(currentSize, userWorkspaceSettings.enableKDWidgetCustomHeight) ||
```

Comment:
not needed if grid heights are already removed before sending layout to the DashboardLayout